### DISCUSSION TITLE
Black screen on the software interface (MacOS)

### DISCUSSION THREAD

**Discussion by xiaojinz** [hernanchavezthielemann/GRO2LAM#45] [Q&A] [answered]
**Describe the bug**
After the software is installed, the interface is just a black screen. 

**Expected behavior**
I am wondering if there is any requirement for GUI compatibility?

**Screenshots**
The following is my screenshot for the GRO2LAM interface. 
![Screen Shot 2022-04-18 at 8 04 45 PM](https://user-images.githubusercontent.com/89263637/163895717-00c2dd07-c8eb-4112-94e9-d2d3e4a0db71.png)

**Desktop (please complete the following information):**
 - Python 3.8.8

>> Try to isolate the issue ;)

**Comment by hernanchavezthielemann** [comment, marked as answer]
Hi Xiao,

That seems like a Tkinter refresh related problem, maybe something with your platform, is that MacOS?
I haven't tested it on mac, so I have no clue about possible behavior issues there.

Nevertheless, have you tried to grab the window from the top banner with your mouse and shake it?

after a quick search, seems like there are some issues on mac, but there are solutions, look for stuff like:
https://github.com/marcelotduarte/cx_Freeze/issues/849

best regards,
Hernan

**Comment by hernanchavezthielemann** [comment]
So please let me know how it goes @xiaojinz

**Comment by xiaojinz** [comment]
> marcelotduarte/cx_Freeze#849

Dear Hernan,

Thank you very much for the information. The link you shared is very helpful! Yes, I use Mac OS. I just fixed the black screen issue by updating my Python package. Now it works well. Thank you and have a good day!

Best wishes,
Xiaojin

**Comment by hernanchavezthielemann** [comment]
Hello Xiaojin,

Great, thank you for letting me know. It is good to know.
I hope you find then the software useful :).

Best wishes,
Hernan.
